server/src/score/score.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Application } from '../application/application.entity';
+import { ScoreBaseEntity } from '../common/score-base.entity';
+import { User } from '../user/user.entity';
+import { Entity, Column, PrimaryGeneratedColumn, OneToOne, JoinColumn } from 'typeorm';
+
+@Entity({
+  name: 'pbgp_score',
+})
+export class Score extends ScoreBaseEntity {
```

Comment:
This can be BroaderReviewScore or something. Also, the service method names should diff between this table and WorkshopReviewScore table

Reply:
Renamed to BroaderReviewScore